### PR TITLE
Update recipe for py312_patch

### PR DESCRIPTION
**Pull request by cbouss**
Preparation for py312 release.

### CODE REVIEW

**Review by JeanChristopheMorinPerso — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ source:
 
 build:
   number: {{ build }}
-  script: python setup.py install --single-version-externally-managed --record=record.txt
+  script: pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:22]: I would add `pip` to `host`, and `pip check` to `tests`. It's up to you

Comment (skupr-anaconda) [reply recipe/meta.yaml:22]: The home ULR isn't working, so update to https://github.com/linsomniac/python-memcached

Comment (skupr-anaconda) [reply recipe/meta.yaml:22]: license PSF-2.0

Comment (cbouss) [reply recipe/meta.yaml:22]: addressed, thank you!

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**